MaxMind.Db/MemoryMapBuffer.cs

```diff
@@ -31,15 +31,24 @@ private MemoryMapBuffer(string file, FileInfo fileInfo) : base((int)fileInfo.Len
                 {
                     _memoryMappedFile = MemoryMappedFile.OpenExisting(mapName, MemoryMappedFileRights.Read);
                 }
+#if !NETSTANDARD1_4
                 catch (Exception ex) when (ex is IOException || ex is NotImplementedException)
+#else
+                catch (Exception ex) when (ex is IOException || ex is NotImplementedException || ex is PlatformNotSupportedException)
+#endif
                 {
-                    using (
-                        var stream = new FileStream(file, FileMode.Open, FileAccess.Read,
-                            FileShare.Delete | FileShare.Read))
-                    {
-                        _memoryMappedFile = MemoryMappedFile.CreateFromFile(stream, mapName, Length,
+                    var stream = new FileStream(file, FileMode.Open, FileAccess.Read,
+                                                FileShare.Delete | FileShare.Read);
+#if !NETSTANDARD1_4
```

Comment:
Added a comment, and the reason why we can't use the six-param signature with Net45 (it was introduces in .NET46 and in NetStandard1.0, the overload with seven-param is removed).